pkg/controller/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -427,29 +427,23 @@ func (r *ReconcileCredentialsRequest) Reconcile(request reconcile.Request) (reco
 		return reconcile.Result{}, err
 	}
 
-	// Ensure we have the controller reference set on the managed secret if it exists:
-	// TODO: we could refactor a bit so this is set when it's created, but it's done in sync.go which currently doesn't know about the full CR.
-	secret := &corev1.Secret{}
-	err = r.Client.Get(context.TODO(), types.NamespacedName{Namespace: cr.Spec.SecretRef.Namespace, Name: cr.Spec.SecretRef.Name}, secret)
-	if err != nil {
-		logger.WithError(err).Error("error looking up secret")
-		return reconcile.Result{}, err
-	}
-
-	return reconcile.Result{}, nil
+	return reconcile.Result{}, syncErr
 }
 
 func (r *ReconcileCredentialsRequest) updateActuatorConditions(cr *minterv1.CredentialsRequest, reason minterv1.CredentialsRequestConditionType, conditionError error) {
 
-	// first unset any previous conditions
-	setInsufficientCredsCondition(cr, false)
-	setFailedToProvisionCredentialsRequest(cr, false, nil)
```

Comment:
Fixed now, problem was these two lines causing condition timestamps to flap even though the same error was occurring over and over.